pyproject.toml

```diff
@@ -1,12 +1,188 @@
+[tool.cibuildwheel]
+build-verbosity = 2
+
+# Note: Tests are run within a docker container on Linux builds
+# This will have a different
+test-requires = "pytest scipy dask"
+# Import-mode=append to use installed (wheel) version of pyfftw
+# rather than the local module
+test-command = "pytest --import-mode=append {project}"
+
+
+    [tool.cibuildwheel.environment]
+    PIP_PREFER_BINARY = true
+
+    [tool.cibuildwheel.windows]
+    archs = [
+        "AMD64",
+    ]
+
+    [tool.cibuildwheel.macos]
+
+    # See: https://github.com/pyFFTW/pyFFTW/issues/324
+    test-skip = "*"
+
+    archs = [
+        "x86_64",
+
+        # Unfortunately we don't have Apple Silicon support yet
+        # TODO: Fix this
+        # Current problem seems to be that installed libfftw3 does not provide arm64
+        # symbols
+        # "universal2",
+        # "arm64",
+    ]
+
+        [tool.cibuildwheel.macos.environment]
+        CC = "/usr/bin/clang"
+        CXX = "/usr/bin/clang++"
+        CFLAGS = "-Wno-implicit-function-declaration -I/usr/local/opt/fftw/include"
+        LDFLAGS = "-Wl,-S,-rpath,/usr/local/opt/fftw/lib -L/usr/local/opt/fftw/lib"
+
+    # On Linux we need to use different package managers depending on image used
+    [tool.cibuildwheel.linux]
+
+    # See: https://github.com/pyFFTW/pyFFTW/issues/325
+    # See: https://github.com/pyFFTW/pyFFTW/issues/326
+    test-skip = [
+        "*i686",
+        "*aarch64",
+    ]
+
+    test-command = "ls -la; echo $PYTHONPATH; pytest -v --import-mode=append {project}/tests"
```

Comment:
Did you try the tests on MacOS using this form of `test-command`? It differs from the one up top in that it lists `{project}/tests` instead of `{project}`!

Reply:
Whoops - that was debugging and meant to have been removed. Good catch.

I believe I tried both versions and that `{project}/tests` should resolve to the same test-suite. Will have a look at this now.

Reply:
Running CI again now having removed that. If Linux builds fail then this was likely the issue. Thank you for catching this!

Reply:
This has been resolved. See #324, #327